config/locales/en.yml

```diff
@@ -332,6 +332,7 @@ en:
     add_to_case_history: Add to case history
     assign: Assign to %{business_unit_name}
     move: Move to this directorate
```

Comment:
Again for accessiblity in these can you please say "Move xxx" or "Move to yyy" or "Move xxx to yyy" - basically the links need to make sense in isolation if possible. I think personally that "Move to yyy" is enough because the page is already of xxx so the user knows.

Reply:
Not very sure whether we need to redesign if we change the name of the button, the following are the screenshot for the buttons 
e.g. - move business unit
<img width="273" alt="Screenshot 2021-05-28 at 23 45 05" src="https://user-images.githubusercontent.com/26111671/120048370-d1640400-c00e-11eb-8a1a-1b400d912a67.png">
- move directorate
<img width="333" alt="Screenshot 2021-05-28 at 23 43 34" src="https://user-images.githubusercontent.com/26111671/120048391-de80f300-c00e-11eb-900a-8391955938f1.png">

Reply:
We need to change the wording only e.g. "Move to Private Office" or "Move to testing-group" - so the blind person can hear this.

Reply:
done